### PR TITLE
Add grenade spinner

### PR DESCRIPTION
**Pull request by onury**
grenade-2

### CODE REVIEW

**Comment by sindresorhus** [comment]
This moves the text:

**Comment by onury** [comment]
@sindresorhus fixed.

File: spinners.json [hunk 1]
```diff
@@ -867,18 +867,34 @@
...
+			"′   ",
+			" ´ ",
+ 			" ‾ ",
+			"  ⸌",
+			"  ⸊",
+			"  |",
+			"  ⁎",
+			"  ⁕",
+			" ෴ ",
+			"  ⁓"
```
Comment (sindresorhus) [inline spinners.json]: I think we should add maybe 3 empty frames here. This animation needs a short pause before replaying.

```json
			"   ",
			"   ",
			"   "
```

Comment (onury) [reply spinners.json]: yeah, looks better now.

**Pull request merged**